bin/build-js.js

```diff
@@ -21,19 +21,16 @@ function buildUppyBundle (minify) {
   var bundleFile = minify ? 'uppy.min.js' : 'uppy.js'
 
   var b = browserify(src, { debug: true, standalone: 'Uppy' })
-  if (minify) {
-    b.plugin('minifyify', {
-      map: bundleFile + '.map',
-      output: path.join(distPath, bundleFile + '.map')
-    })
-  }
-  // b.transform(yoyoify)
+  b.plugin(commonShakeify)
+  b.plugin(packFlat)
   b.transform(babelify)
   b.on('error', handleErr)
 
   return new Promise(function (resolve, reject) {
     if (minify) {
       b.bundle()
+      .pipe(uglify())
```

Comment:
Minifying the final result instead of the individual modules … common-shakeify only comments things, so minification needs to happen after common-shakeify.